src/Containers.jsx

```diff
@@ -28,112 +28,266 @@ import { PodActions } from './PodActions.jsx';
 
 const _ = cockpit.gettext;
 
-class Containers extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            selectContainerDeleteModal: false,
-            selectContainerCheckpointModal: false,
-            selectContainerRestoreModal: false,
-            setContainerRemoveErrorModal: false,
-            containerWillDelete: {},
-            containerWillCheckpoint: {},
-            containerWillRestore: {},
-            checkpointInProgress: false,
-            restoreInProgress: false,
-            width: 0,
-        };
-        this.renderRow = this.renderRow.bind(this);
-        this.onWindowResize = this.onWindowResize.bind(this);
-        this.restartContainer = this.restartContainer.bind(this);
-        this.startContainer = this.startContainer.bind(this);
-        this.stopContainer = this.stopContainer.bind(this);
-        this.deleteContainer = this.deleteContainer.bind(this);
-        this.handleCancelContainerDeleteModal = this.handleCancelContainerDeleteModal.bind(this);
-        this.handleCheckpointContainerDeleteModal = this.handleCheckpointContainerDeleteModal.bind(this);
-        this.handleRestoreContainerDeleteModal = this.handleRestoreContainerDeleteModal.bind(this);
-        this.handleRemoveContainer = this.handleRemoveContainer.bind(this);
-        this.handleCheckpointContainer = this.handleCheckpointContainer.bind(this);
-        this.handleRestoreContainer = this.handleRestoreContainer.bind(this);
-        this.handleCancelRemoveError = this.handleCancelRemoveError.bind(this);
-        this.handleForceRemoveContainer = this.handleForceRemoveContainer.bind(this);
-
-        this.cardRef = React.createRef();
-
-        window.addEventListener('resize', this.onWindowResize);
-    }
-
-    componentDidMount() {
-        this.onWindowResize();
-    }
-
-    componentWillUnmount() {
-        window.removeEventListener('resize', this.onWindowResize);
-    }
-
-    deleteContainer(container, event) {
+const ContainerActions = ({ container, onAddNotification, version }) => {
+    const [removeErrorModal, setRemoveErrorModal] = useState(false);
+    const [deleteModal, setDeleteModal] = useState(false);
+    const [checkpointInProgress, setCheckpointInProgress] = useState(false);
+    const [checkpointModal, setCheckpointModal] = useState(false);
```

Comment:
checkpoint and restore should be separate group in the dropdown as well (separated by line)
![Screenshot from 2021-09-08 09-49-14](https://user-images.githubusercontent.com/12330670/132468687-4612442c-3183-4df6-86b8-5fe8009f8045.png)

Reply:
Fixed.